src/main/groovy/com/bmuschko/gradle/cargo/CargoBasePlugin.groovy

```diff
@@ -31,7 +31,7 @@ import org.gradle.api.Task
 class CargoBasePlugin implements Plugin<Project> {
     static final String CONFIGURATION_NAME = 'cargo'
     static final String DAEMON_CONFIGURATION_NAME = 'cargoDaemon'
-    static final String CARGO_DEFAULT_VERSION = '1.6.8'
+    static final String CARGO_DEFAULT_VERSION = '1.7.7'
```

Comment:
I'd suggest you upgrade to Codehaus Cargo 1.7.9, which has a second fix for https://codehaus-cargo.atlassian.net/browse/CARGO-1482 and is beneficial for the Gradle Cargo plugin

Reply:
@alitokmen Is this change backward-compatible with older versions of Ant?

Reply:
Yes, it is - We have a static method that checks for exceptions and tries both ways